suites/reef/nvmeof/tier-2_2-nvmeof-gw_16-sub_ns.yaml

```diff
@@ -0,0 +1,184 @@
+# Test suite to scale with 2 GWs having 2 subsystem with 400 namespaces
```

Comment:
Looks like the suite description is not in sync.

Reply:
Thanks. I have updated it @HaruChebrolu